.coderabbit.yaml

```
@@ -0,0 +1,18 @@
+      해당 코드 스타일 가이드를 따릅니다:
+      - 테스트 메소드명은 `should + 동작 + 조건`으로 일관된 네이밍 유지
+  request_changes_workflow: true # CodeRabbit 리뷰 후 PR Approve 진행
+  high_level_summary: true
+  review_status: true
+  collapse_walkthrough: false
+  poem: false
+  auto_review:
+    enabled: true
+    drafts: false
+chat:
+  auto_reply: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)